SPECS-CROSS/binutils_cross/binutils_cross.spec

```diff
@@ -0,0 +1,248 @@
+# Globals which should be in a macro file.
+# These should be set programatically in the future.
+%global _host_arch      x86_64
+%global _target_arch    aarch64
+%global _tuple          %{_target_arch}-%{_vendor}-linux-gnu
+%global _cross_name     %{_target_arch}-%{_vendor}-linux-gnu
+
+# Folders which should be in our macro file
+%global _opt                /opt/
```

Comment:
Why are these files installed in the /opt folder?

opt is traditionally for 3rd party software.

https://linuxhandbook.com/linux-directory-structure/#:~:text=%2Fopt%20%E2%80%93%20Optional%20software%20Traditionally%2C%20the%20%2Fopt%20directory,so%20that%20all%20the%20users%20can%20run%20it.